### ISSUE TITLE
build local testnet

### ISSUE THREAD

**Issue by zyDevoted** [sigp/lighthouse#4771]
## Description

(info: Execution endpoint does not support Capella methods: Required Methods Unsupported: engine_getPayloadV2 engine_forkchoiceUpdatedV2 engine_newPayloadV2, hint: try updating the execution endpoint, service: slot_notifie)

## Version

lighthouse:Lighthouse v4.4.1-2841f60
rust:cargo 1.72.1 (103a7ff2e 2023-08-15)

## Present Behaviour

In [Manual creation of local testnet](https://github.com/sigp/lighthouse/tree/unstable/scripts/local_testnet#manual-creation-of-local-testnet)
when I run the beacon node after the geth node, I received the info "Not ready for Capella                   info: Execution endpoint does not support Capella methods: Required Methods Unsupported: engine_getPayloadV2 engine_forkchoiceUpdatedV2 engine_newPayloadV2, hint: try updating the execution endpoint, service: slot_notifier" and the geth can not sync and says Served engine_exchangeCapabilities       conn=127.0.0.1:40610 reqid=1 duration="3.357µs"  err="the method engine_exchangeCapabilities does not exist/is not available"

## Expected Behaviour

I don't know the expected behaviour of this operation. Please help me.

**Comment by zyDevoted** [comment]
my geth version is Version: 1.13.1-stable
Git Commit: 3f40e65c484486dea6cff80b7db178985d21a2c6

**Comment by chong-he** [comment]
Thanks for the report. Reopening this issue as this is a valid issue and reproducible. 

We found a few issues with manually starting the testnet:

1. The port used in the example 
```
./beacon_node.sh $HOME/.lighthouse/local-testnet/node_1 9000 8000 http://localhost:6000 ~/.lighthouse/local-testnet/geth_1/geth/jwtsecret
```
is the wrong port to connect to geth. In the example given, it should be 7000 instead.

2. The genesis.json is not updated with Shanghai/Capella time. This is fixed with having another file `setup_time.sh` 

The fix is here (note that I revised the ports used so that it is consistent with the one when running [start_local_testnet.sh](https://github.com/sigp/lighthouse/blob/stable/scripts/local_testnet/start_local_testnet.sh):
#4749

**Comment by jimmygchen** [comment]
Fixed in #4749 🎉
